script/build-app.sh

```diff
@@ -52,9 +52,10 @@ BUILDS=builds
 RELEASE="$NAME-$VERSION-$OS"
 RELEASE_DIR="$BUILDS/$RELEASE"
 RELEASE_TARBALL="$BUILDS/${RELEASE}.tar.gz"
+RELEASE_ZIP="${RELEASE}.zip"
```

Comment:
Can we use $BUILDS here and get rid of the pushd and popd? When any paths change it's helpful to only have to change one variable instead of multiple locations in the script